toc.md

```diff
@@ -700,6 +858,14 @@ module Workerholic
 end
 ```
 
+In the code above the data range would be indicated by having the earliest timestamp of the data range as the `minscore` and the latest timestamp as the `maxscore`.
```

Comment:
In the code above, the data range would be delimited by having the earliest timestamp of the data range as the `minscore` and the latest timestamp as the `maxscore`.